docs/api/role-management/put.asciidoc

```diff
@@ -82,7 +82,7 @@ PUT /api/security/role/my_kibana_role
         "dashboard": [
           "all"
         ],
-        "dev_tools": [
+        "devTools": [
```

Comment:
Please do not rename feature and role id - those could cause compatibility problems. This is just about the plugin id of the NP plugin